### PR TITLE
add link to mapa from menu and set geo Belarus

### PR DESCRIPTION
**Pull request by tragladit**

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -26,9 +26,10 @@ import mapConst             from "../constants/Map"
 class Main extends Component {
   
   render(){
+    
     const { dispatch, search, view, server, map, form, url, user, t } = this.props;
     const { entries, ratings } = server;
-
+    console.log(map)
```
Comment (fr0zen) [inline src/components/App.js]: @tragladit May you explain this string a little bit?

Comment (tragladit) [reply src/components/App.js]: @fr0zen this string not needed. Forgot to delete ))

Comment (fr0zen) [reply src/components/App.js]: @tragladit please remove this line and commit, changes must appear here)

**Comment by rizomaa** [comment]
@fr0zen does this PR need some changes?

**Pull request merged**